HHS/OPRE-OPS
# feat: :sparkles: toggle all budget lines

## What changed

feat: ✨ toggle all budget lines
- renames AgreementBLIReviewTable to BLIReviewTables and moves the BudgetLineItems directory
- adds slot for Table component to allow for editing the first heading

## Issue

#1558 

## How to test

1. Goto Agreements list and select Agreement 1 for review
2. Select DRAFT to Planning
3. See BLITable and select main toggle in heading
4. All DRAFT budgetlines should be checked
5. UnCheck main toggle
6. All DRAFT budgetlines should be un-checked

## Screenshots

<img width="1268" alt="image" src="https://github.com/HHS/OPRE-OPS/assets/4629398/53d2863c-3277-4372-9d84-14d8d6e82be5">

## Definition of Done Checklist
